### PR TITLE
Feature dscov remote render

### PR DESCRIPTION
**Pull request by kmilo9999**
- Added yml for remote rendering workshop
- Added visualization category

### CODE REVIEW

**Comment by kmilo9999** [comment]
resolves #143

File: our-work/workshops-and-talks/remote_renderer [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+languages:
+  - unix-cli
+tags:
+  - visualization
+  - oscar
+people:
+  - name: Camilo Diaz
+    github_user: kmilo9999
+groups:
+  - 3d-visualization-workshop
```
Comment (eldu) [inline our-work/workshops-and-talks/remote_renderer:13]:
```suggestion
  - dscov
```

**Review by eldu — CHANGES_REQUESTED** [review]
For some reason, I'm not seeing this talk on the preview link for the website?

I added in a comment to change the group maybe that will help?

**Comment by kmilo9999** [comment]
@eldu I applied the suggested change , but it's still not there. Any thoughts?

**Comment by eldu** [comment]
> @eldu I applied the suggested change , but it's still not there. Any thoughts?

hmm no idea, i'll ask in code buddies maybe someone can find something i'm not seeing?

File: our-work/workshops-and-talks/remote_renderer [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - oscar
+people:
+  - name: Camilo Diaz
+    github_user: kmilo9999
+groups:
+  - dscov
+links:
+  - category: video 
+    url: https://brown.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=526cb8b8-2ce3-439b-ab84-adc001569019
+  - category: documentation
```
Comment (hollandjg) [inline our-work/workshops-and-talks/remote_renderer:17]: there isn't a "documentation" category in the enum – `enum('code', 'slides', 'website', 'video')`

Comment (kmilo9999) [reply our-work/workshops-and-talks/remote_renderer:17]: How should I categorize the documentation ? Is website ok?  The url points to the remote render in oscar documentation

Comment (mirestrepo) [reply our-work/workshops-and-talks/remote_renderer:17]: We can just add `documentation` to the enum

Comment (mirestrepo) [reply our-work/workshops-and-talks/remote_renderer]: Actually, John fixed this in his PR, which I just merged

Comment (mirestrepo) [reply our-work/workshops-and-talks/remote_renderer]: @hollandjg @eldu should we leave the tag as `website` or add `documentation` to the enum. I'm not sure which makes more sense

Comment (eldu) [reply our-work/workshops-and-talks/remote_renderer]: I think either would work !

**Review by eldu — APPROVED** [review]

**Review by hollandjg — APPROVED** [review]
Looks good to me!

**Pull request merged**